privacyidea/static/components/user/views/user.add.html

```diff
@@ -1,5 +1,15 @@
+<style type="text/css">
+  .css-form input.ng-invalid.ng-touched {
+    background-color: #FA787E;
+  }
+
+  .css-form input.ng-valid.ng-touched {
+    background-color: #78FA89;
+  }
+</style>
+
 <div class="panel" ng-controller="userAddController">
-    <form name="formAddUser" role="form" validate>
+    <form name="User" role="form" novalidate>
```

Comment:
I think there is no need to change the name of the form. I would leave it as "formAddUser", since this reflects the naming of other templates.